packages/front-end/components/Radix/Tabs.tsx

```diff
@@ -100,6 +101,34 @@ export const TabsList = forwardRef<HTMLDivElement, TabsListProps>(
   }
 );
 
+type StickyTabsListProps = { pinnedClass?: string } & TabsListProps;
+
+export const StickyTabsList = forwardRef<HTMLDivElement, StickyTabsListProps>(
+  function StickyTabsList({ pinnedClass = "pinned", ...props }, ref) {
+    // NB: Keep in sync with .experiment-tabs top property in global.scss
+    const TABS_HEADER_HEIGHT_PX = 55;
+    const tabsRef = useRef<HTMLDivElement>(null);
+    const [headerPinned, setHeaderPinned] = useState(false);
+    const { scrollY } = useScrollPosition();
+    useEffect(() => {
+      if (!tabsRef.current) return;
+      const isHeaderSticky =
+        tabsRef.current.getBoundingClientRect().top <= TABS_HEADER_HEIGHT_PX;
+      setHeaderPinned(isHeaderSticky);
+    }, [scrollY]);
```

Comment:
Running this on every scroll change is probably expensive.  Might be better to use an IntersectionObserver.  Even better if we can get rid of the need for a `pinnedClass` entirely and have one set of styles that works for both pinned and unpinned.

Reply:
I used IntersectionObserver, there was no good way to remove the sticky class entirely given our design.